<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralNoticeOfHearingDialog</class>
 <widget class="QDialog" name="GeneralNoticeOfHearingDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1088</width>
    <height>737</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>2500</width>
    <height>3500</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>29</red>
        <green>61</green>
        <blue>102</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>29</red>
        <green>61</green>
        <blue>102</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>29</red>
        <green>61</green>
        <blue>102</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>29</red>
        <green>61</green>
        <blue>102</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>29</red>
        <green>61</green>
        <blue>102</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>29</red>
        <green>61</green>
        <blue>102</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>29</red>
        <green>61</green>
        <blue>102</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>29</red>
        <green>61</green>
        <blue>102</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>29</red>
        <green>61</green>
        <blue>102</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font">
   <font>
    <family>Palatino Linotype</family>
    <pointsize>10</pointsize>
    <weight>75</weight>
    <italic>false</italic>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>General Notice of Hearing Information</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(29, 61, 102);
font: 75 10pt &quot;Palatino Linotype&quot;;
font-weight: bold;</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_8">
   <item row="0" column="0">
    <widget class="QFrame" name="frame_4">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(29, 61, 102);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QScrollArea" name="scrollArea">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(12, 117, 255);</string>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>1048</width>
           <height>697</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="2" column="0">
           <widget class="QFrame" name="frame_8">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(12, 117, 255);</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_10">
             <item row="2" column="0">
              <widget class="QPushButton" name="close_dialog_Button">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 96, 82);
font: 75 11pt &quot;Palatino Linotype&quot;;
font-weight: bold;</string>
               </property>
               <property name="text">
                <string>Close Dialog</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QPushButton" name="create_entry_Button">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(0, 255, 0);
font: 75 11pt &quot;Palatino Linotype&quot;;
font-weight: bold;</string>
               </property>
               <property name="text">
                <string>Open Entry</string>
               </property>
               <property name="autoDefault">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QFrame" name="frame_5">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(235, 235, 235);</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="lineWidth">
             <number>2</number>
            </property>
            <layout class="QGridLayout" name="gridLayout_6">
             <item row="0" column="0">
              <layout class="QGridLayout" name="gridLayout_3" rowstretch="1,0,0,0,0">
               <property name="verticalSpacing">
                <number>24</number>
               </property>
               <item row="3" column="0">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Hearing Time:</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="hearing_date_label">
                 <property name="text">
                  <string>Hearing Date:</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="NoScrollDateEdit" name="hearing_dateEdit">
                 <property name="styleSheet">
                  <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
                 </property>
                 <property name="calendarPopup">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="0" column="0" colspan="2">
                <widget class="QLabel" name="label_5">
                 <property name="font">
                  <font>
                   <family>Palatino Linotype</family>
                   <pointsize>10</pointsize>
                   <weight>75</weight>
                   <italic>false</italic>
                   <bold>true</bold>
                   <underline>true</underline>
                  </font>
                 </property>
                 <property name="text">
                  <string>SCHEDULING INFORMATION</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="hearing_type_label">
                 <property name="text">
                  <string>Hearing Type:</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="NoScrollComboBox" name="hearing_time_box">
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(255, 255, 255);</string>
                 </property>
                 <item>
                  <property name="text">
                   <string>8:00 AM</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>8:15 AM</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>8:30 AM</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>8:45 AM</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>9:00 AM</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>9:15 AM</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>9:30 AM</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>9:45 AM</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>10:00 AM</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>10:15 AM</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>10:30 AM</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>10:45 AM</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>11:00 AM</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>11:15 AM</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>11:30 AM</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>1:00 PM</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>1:15 PM</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>1:30 PM</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>1:45 PM</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>2:00 PM</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>2:15 PM</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>2:30 PM</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>2:45 PM</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>3:00 PM</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>3:15 PM</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>3:30 PM</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>3:45 PM</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>4:00 PM</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>4:15 PM</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="4" column="0">
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>Hearing Location:</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="1">
                <widget class="NoScrollComboBox" name="hearing_location_box">
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(255, 255, 255);</string>
                 </property>
                 <item>
                  <property name="text">
                   <string/>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Courtroom A</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Courtroom B</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Courtroom C</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="NoScrollComboBox" name="hearing_type_box">
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(255, 255, 255);</string>
                 </property>
                 <property name="editable">
                  <bool>false</bool>
                 </property>
                 <item>
                  <property name="text">
                   <string>Plea Hearing</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Motion to Suppress Hearing</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Oral Hearing</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Seal Record</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Motion Hearing</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QFrame" name="case_name_Frame">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(240, 240, 240);</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="lineWidth">
             <number>2</number>
            </property>
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="0" column="0">
              <layout class="QGridLayout" name="gridLayout" columnstretch="1,2,1,2,1">
               <property name="verticalSpacing">
                <number>24</number>
               </property>
               <item row="1" column="0">
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Def. First Name:</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLineEdit" name="defendant_first_name_lineEdit">
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(255, 255, 255);</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <widget class="QLabel" name="label_26">
                 <property name="text">
                  <string>Date:</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="3">
                <widget class="NoScrollDateEdit" name="entry_date">
                 <property name="focusPolicy">
                  <enum>Qt::NoFocus</enum>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
                 </property>
                 <property name="minimumDate">
                  <date>
                   <year>2021</year>
                   <month>1</month>
                   <day>1</day>
                  </date>
                 </property>
                 <property name="calendarPopup">
                  <bool>true</bool>
                 </property>
                 <property name="date">
                  <date>
                   <year>2021</year>
                   <month>1</month>
                   <day>1</day>
                  </date>
                 </property>
                </widget>
               </item>
               <item row="1" column="4">
                <widget class="QPushButton" name="cancel_Button">
                 <property name="focusPolicy">
                  <enum>Qt::NoFocus</enum>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(160, 160, 160);</string>
                 </property>
                 <property name="text">
                  <string>Cancel</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Def. Last Name:</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QLineEdit" name="defendant_last_name_lineEdit">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(255, 255, 255);</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="2">
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Case Number:</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="3">
                <widget class="QLineEdit" name="case_number_lineEdit">
                 <property name="maximumSize">
                  <size>
                   <width>500</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(255, 255, 255);</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="4">
                <widget class="QPushButton" name="clear_fields_case_Button">
                 <property name="focusPolicy">
                  <enum>Qt::NoFocus</enum>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(160, 160, 160);</string>
                 </property>
                 <property name="text">
                  <string>Clear Fields</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="label_24">
                 <property name="text">
                  <string>Def. Counsel:</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="DefenseCounselComboBox" name="defense_counsel_name_box">
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(255, 255, 255);</string>
                 </property>
                 <property name="editable">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="0" column="0" colspan="5">
                <widget class="QLabel" name="label_6">
                 <property name="font">
                  <font>
                   <family>Palatino Linotype</family>
                   <pointsize>10</pointsize>
                   <weight>75</weight>
                   <italic>false</italic>
                   <bold>true</bold>
                   <underline>true</underline>
                  </font>
                 </property>
                 <property name="text">
                  <string>CASE INFORMATION</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NoScrollDateEdit</class>
   <extends>QDateEdit</extends>
   <header>munientry.widgets.custom_widgets</header>
  </customwidget>
  <customwidget>
   <class>NoScrollComboBox</class>
   <extends>QComboBox</extends>
   <header>munientry.widgets.combo_boxes</header>
  </customwidget>
  <customwidget>
   <class>DefenseCounselComboBox</class>
   <extends>QComboBox</extends>
   <header>munientry.widgets.combo_boxes</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>hearing_type_box</tabstop>
  <tabstop>hearing_dateEdit</tabstop>
  <tabstop>hearing_time_box</tabstop>
  <tabstop>hearing_location_box</tabstop>
  <tabstop>defendant_first_name_lineEdit</tabstop>
  <tabstop>defendant_last_name_lineEdit</tabstop>
  <tabstop>defense_counsel_name_box</tabstop>
  <tabstop>case_number_lineEdit</tabstop>
  <tabstop>create_entry_Button</tabstop>
  <tabstop>close_dialog_Button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <slots>
  <slot>create_entry()</slot>
  <slot>continue_dialog()</slot>
  <slot>close_window()</slot>
  <slot>set_dialog()</slot>
  <slot>update_case_information()</slot>
  <slot>ovi_dialog()</slot>
  <slot>sentencing_dialog()</slot>
  <slot>proceed_to_sentencing()</slot>
  <slot>proceed_to_ovi()</slot>
  <slot>add_charge()</slot>
  <slot>set_statute()</slot>
  <slot>set_pay_date()</slot>
  <slot>amend_offense()</slot>
  <slot>set_offense()</slot>
  <slot>add_charge_to_view()</slot>
  <slot>delete_charge()</slot>
  <slot>delete_charge_from_view()</slot>
  <slot>close_event()</slot>
  <slot>set_fra_in_file()</slot>
  <slot>set_fra_in_court()</slot>
  <slot>start_add_conditions_dialog()</slot>
  <slot>start_amend_offense_dialog()</slot>
 </slots>
</ui>
